certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/Main.kt

```diff
@@ -165,16 +182,15 @@ class Main : Callable<Int> {
   companion object {
     internal const val NAME = "cft"
 
-    val confDir = File(System.getProperty("user.home"), ".cft").also {
-      if (!it.isDirectory) {
```

Comment:
why remove this?

Reply:
Trying to use Path API exclusively, and at this point we don't know the Filesystem.